test/e2e/logging_test.go

```diff
@@ -5,84 +5,78 @@ import (
 
 	operatorsv1 "github.com/openshift/api/operator/v1"
 	deploymentsub "github.com/openshift/console-operator/pkg/console/subresource/deployment"
-	"github.com/openshift/console-operator/pkg/testframework"
+	"github.com/openshift/console-operator/test/e2e/framework"
 )
 
-func setupLoggingTestCase(t *testing.T) *testframework.Clientset {
-	client := testframework.MustNewClientset(t, nil)
-	testframework.MustManageConsole(t, client)
-	testframework.MustNormalLogLevel(t, client)
-	return client
+func setupLoggingTestCase(t *testing.T) (*framework.ClientSet, *operatorsv1.Console) {
+	return framework.StandardSetup(t)
 }
 
-func cleanUpLoggingTestCase(t *testing.T, client *testframework.Clientset) {
-	testframework.WaitForSettledState(t, client)
+func cleanUpLoggingTestCase(t *testing.T, client *framework.ClientSet) {
+	framework.StandardCleanup(t, client)
 }
 
 // TestDebugLogLevel sets 'Debug' LogLevel on the console operator and tests
 // if '--log-level=*=DEBUG' flag is set on the console deployment
 func TestDebugLogLevel(t *testing.T) {
-	client := setupLoggingTestCase(t)
-	defer testframework.SetLogLevel(t, client, operatorsv1.Normal)
+	client, _ := setupLoggingTestCase(t)
+	defer cleanUpLoggingTestCase(t, client)
```

Comment:
can't we just call directly ?
```suggestion
	defer framework.StandardCleanup(t, client)
```

Reply:
interim atm, see above https://github.com/openshift/console-operator/pull/248/files#r307366710